src/pages/Cart/Cart.js

```diff
@@ -52,24 +54,28 @@ export default class Cart extends Component {
   };
 
   handleIncrement = cart => {
-    const cartList = this.state.cartList.map(item => {
-      if (item.option_id === cart.option_id) {
-        const quantity = cart.quantity + 1;
-        return { ...cart, quantity: quantity >= 99 ? 99 : quantity };
-      }
-      return item;
-    });
-    this.setState({ cartList });
-    const requestOptions = {
-      method: 'PATCH',
-      headers: {
-        Authorization: localStorage.getItem('TOKEN'),
-      },
-      body: JSON.stringify({
-        quantity: cart.quantity >= 99 ? 99 : cart.quantity + 1,
-      }),
-    };
-    fetch(`${CART_API}/${cart.option_id}`, requestOptions);
+    if (cart.quantity < cart.stocks) {
+      const cartList = this.state.cartList.map(item => {
+        if (item.option_id === cart.option_id) {
+          const quantity = cart.quantity + 1;
+          return { ...cart, quantity: quantity >= 99 ? 99 : quantity };
+        }
+        return item;
+      });
+      this.setState({ cartList });
+      const requestOptions = {
+        method: 'PATCH',
+        headers: {
+          Authorization: localStorage.getItem('TOKEN'),
+        },
+        body: JSON.stringify({
+          quantity: cart.quantity >= 99 ? 99 : cart.quantity + 1,
+        }),
+      };
+      fetch(`${CART_API}/${cart.option_id}`, requestOptions);
```

Comment:
```suggestion
      });
      
      this.setState({ cartList });
      
      const requestOptions = {
        method: 'PATCH',
        headers: {
          Authorization: localStorage.getItem('TOKEN'),
        },
        body: JSON.stringify({
          quantity: cart.quantity >= 99 ? 99 : cart.quantity + 1,
        }),
      };
      
      fetch(`${CART_API}/${cart.option_id}`, requestOptions);
```
- 구분되는 로직들은 한줄 씩 띄워주는게 좋겠습니다.

Reply:
> * 구분되는 로직들은 한줄 씩 띄워주는게 좋겠습니다.

구분을 위해 띄어쓰기 완료했습니다.